src/Message/IPNCallback.php

```diff
@@ -116,6 +116,18 @@ public function isAuthOnlySingleCharge()
         return $this->getIPNType() === 'AUTH_ONLY_SINGLE_CHARGE';
     }
 
+    /**
+     * Is this a AUTH_ONLY_SINGLE_CHARGE IPN? This IPN is fired when
+     * a zero dollar transaction is made for standard subscription with trial.
```

Comment:
same thing about mentioning the contract. Or better yet, maybe linking to some documentation that discusses this?

Reply:
good call, let me add the url